crates/matrix-sdk/src/client/mod.rs

```diff
@@ -1872,7 +1874,7 @@ impl Client {
             .await?
             .map(|c| c.deserialize())
             .transpose()?
-            .unwrap_or_else(|| ruma::events::direct::DirectEventContent(BTreeMap::new()));
+            .unwrap_or_else(|| DirectEventContent(BTreeMap::new()));
```

Comment:
Actually, with the current Ruma rev, this can be:

```suggestion
            .unwrap_or_default();
```